src/Ocean/OceanProblems/OceanProblems.jl

```diff
@@ -0,0 +1,46 @@
+module OceanProblems
+
+export SimpleBox, Fixed, Rotating, HomogeneousBox, OceanGyre
+
+using StaticArrays
+using CLIMAParameters.Planet: grav
+
+using ...Problems
+
+using ..Ocean
+using ..HydrostaticBoussinesq
+using ..ShallowWater
+using ..SplitExplicit01
+
+import ..Ocean:
+    ocean_init_state!,
+    ocean_init_aux!,
+    kinematic_stress,
+    surface_flux,
+    coriolis_parameter
+
+HBModel = HydrostaticBoussinesqModel
+SWModel = ShallowWaterModel
+
+include("simple_box_problem.jl")
+include("homogeneous_box.jl")
+
+include("shallow_water_initial_states.jl")
+
+function ocean_init_state!(m::SWModel, p::HomogeneousBox, Q, A, localgeo, t)
```

Comment:
In Atmos, we're ordering the parameters as `(problem, balance_law, Q, ...)` -- suggest we use the same ordering.

Reply:
Ah, thanks! This PR didn't touch these function signatures (just moved the function definitions around), so I'll open an issue to address this.